subprojects/gradle/runner/service/src/main/kotlin/com/avito/runner/service/worker/device/adb/AdbDevice.kt

```diff
@@ -434,19 +434,19 @@ data class AdbDevice(
         )
 
     override fun debug(message: String) {
```

Comment:
Let delete  duplicates of the Logger functions and make logger field public

Reply:
agreed; fixed